docs/source/man.rst

```diff
@@ -492,14 +492,14 @@ The ``qpc scan cancel`` command cancels the execution of a scan job. A canceled
 Reports
 --------
 
-Use the ``qpc report`` command to generate a report from a scan. You can generate a report as JavaScript Object Notation (JSON) format or as comma-separated values (CSV) format. There are two different types of report that you can generate, a *detail* report and a *summary* report.
+Use the ``qpc report`` command to generate a report from a scan. You can generate a report as JavaScript Object Notation (JSON) format or as comma-separated values (CSV) format. There are two different types of report that you can generate, a *details* report and a *deployments* report.
 
 
-Viewing the Detail Report
+Viewing the Details Report
 ~~~~~~~~~~~~~~~~~~~~~~~~~
-The ``qpc report detail`` command generates a detailed report that contains the unprocessed facts that are gathered during a scan. These facts are the raw output from network, vcenter, and satellite scans, as applicable.
+The ``qpc report details`` command generates a detailed report that contains the unprocessed facts that are gathered during a scan. These facts are the raw output from network, vcenter, and satellite scans, as applicable.  The ``qpc report details`` command is now deprecated.  The ``qpc report details`` command should be used instead.
```

Comment:
you should change the first ``qpc report details`` back to ``qpc report detail``

Reply:
😂 Indeed.